packages/react/src/transactions/adapters/PendingTransactionGateway/PendingTransactionGateway.ts

```diff
@@ -12,7 +12,7 @@ import {
 } from '@lens-protocol/domain/use-cases/transactions';
 import { assertNever, invariant } from '@lens-protocol/shared-kernel';
 import { IStorage } from '@lens-protocol/storage';
-import differenceBy from 'lodash/differenceBy';
+import differenceBy from 'lodash/differenceBy.js';
```

Comment:
Out of curiosity, are you aware of any plan for Lodash to have proper `exports` configuration?

Reply:
There is `lodash-es` that's under the hood using `es` exports, maybe it's worth to migrate at some point (or completely remove the use of lodash given it's used in just a couple of places)